app/views/shared/_run_table.slim

```diff
@@ -65,5 +61,5 @@
                     = link_to '+', new_rivalry_path(category: run.category)
             - if cols.include?(:uploaded)
               td.align-right.nowrap class=('short' if run.short?) title=run.created_at = "#{time_ago_in_words(run.created_at)} ago"
-    .mx-auto
-      = paginate(runs, theme: 'twitter-bootstrap-4', pagination_class: 'mt-3')
+  .mx-auto
+    = paginate(runs, theme: 'twitter-bootstrap-4', pagination_class: 'mt-3')
```

Comment:
Not sure if this is fixable, but the spacer that is put in the pagination between the last number and then `Next` is not styled liked the rest of the paginator.


<img width="347" alt="screen shot 2018-06-07 at 2 29 31 am" src="https://user-images.githubusercontent.com/7030913/41091141-c09652e4-69fa-11e8-85dd-8054fed8381e.png">

Reply:
Looks intentional by Dashboard, it's not ideal but similar to before I think it's less valuable to change it than the amount of time we'd pay later by trying to restyle the theme.